src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Microsoft.CodeAnalysis.LanguageServer.csproj

```diff
@@ -57,7 +57,7 @@
     <ProjectReference Include="..\Protocol\Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj" />
 
     <!-- Dlls we don't directly reference but need to include to build the MEF composition -->
-    <ProjectReference Include="..\..\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.Features.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.Features.csproj" />
```

Comment:
📝 If we omit `ReferenceOutputAssembly="false"`, the comment needs to be updated. Currently there is a mismatch.

Reply:
I lean towards keeping the exclusion which allows us to make a conscious decision later if we think need to add references in code.

Reply:
> I lean towards keeping the exclusion which allows us to make a conscious decision later if we think need to add references in code.

That's a good point - I've added that line back